_src/_posts/2018-12-19-jplotlib-help-your-teams-to-leverage-data-by-using-the-perfect-graphs.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title:  "How to help your teams to leverage data by using the perfect graphs."
+date:   2018-12-19
+tag:    technology
+categories:
+    - data-science
+    - technology
+    - python
+keywords: "python, matplotlib, visualization, jplotlib"
+author: felipegonzalez
+---
+
+<!--excerpt.start-->
+
+In these times, it is all about Data, yes..
+
+...BUT, it’s not only about having your data available in a clean and easy data-set or feeding a machine learning algorithm, it is also about reading it, sharing it so other people (more or less technical than you) can understand the trends without too many explanations or even better, without feeling that the chart is so over-complicated that they can not even try it.
+
+Because of that, at Jampp, we create our own Matplotlib wrapper, JPlotlib.
+So everyone can use an already customized, Jampp branded and easy to go Python library to introduce data results with no effort.
+
+<!--excerpt.end-->
+
+Before, with an open source library defaults...
+![ open source library default histogram ]({{site.url}}/assets/images/jplotlib/old_histogram.png){: .center-image }
+
+After, with JPlotLib:
+![ jplotlib histogram ]({{site.url}}/assets/images/jplotlib/histogram.png){: .center-image }
+
+You might be thinking: Why? Some libraries already have what you need.
+But that is not true.
+We create and analyze tailored data-sets, user behavior and patterns from millions and millions of data and let’s be honest, almost none of the default suggestions from softwares are the correct ones to show it.
+Some libraries have tons of documentation and bugs, so changing small details requires a lot of manual input, a.k.a “human-time”.
+
+Examples with before and after results and the small coding changes needed in the README are key for an easy and solid experience.
+
+## Extreme-Makeover, Jampp edition
+
+This is the part where we show you how different your visual experience can be by improving your plotting tools.
+
+### Bar Plot
+
+#### Before
+![ basic bar plot ]({{site.url}}/assets/images/jplotlib/old_bar.png){: .center-image }
+
+#### After
+![ jplotlib bar plot ]({{site.url}}/assets/images/jplotlib/bar.png){: .center-image }
+
+### Barh Plot
+
+#### Before
+![ basic barh plot ]({{site.url}}/assets/images/jplotlib/old_barh.png){: .center-image }
+
+#### After
+![ jplotlib barh plot ]({{site.url}}/assets/images/jplotlib/barh.png){: .center-image }
+
+### Scatter Plot
+
+#### Before
+![ basic scatter plot ]({{site.url}}/assets/images/jplotlib/old_scatter.png){: .center-image }
+
+#### After
+![ jplotlib barh plot ]({{site.url}}/assets/images/jplotlib/scatter.png){: .center-image }
+
+
+This wouldn't be complete without some of the houses favorites.
```

Comment:
I think in this case would be "some of the house favorites" :thinking:

Reply:
House's favourites?